src/Microsoft.OData.Core/UriParser/SyntacticAst/RootPathToken.cs

```diff
@@ -0,0 +1,40 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EndPathToken.cs" company="Microsoft">

```

Comment:
Should this be `RootPathToken.cs` instead of `EndPathToken.cs`
```suggestion
// <copyright file="RootPathToken.cs" company="Microsoft">

```

Reply:
#resolved.